backend/README.md

```diff
@@ -52,8 +52,19 @@ cargo install typeshare-cli
 
 6. Start Keycloak
 
+via docker run:
```

Comment:
I have created the client as requested.

* Client ID = `localhost`
* Root URL = `http://localhost:5173/`
* Valid Redirect URLs = `/*`
* Web Origins = `+`

@absurd-turtle , @markus2330 could someone pls verify whether this works? 😄

Reply:
Thank you, following .env works flawlessly for me (including self-registration):

```
DATABASE_URL=postgres://permaplant:permaplant@127.0.0.1/permaplant
BIND_ADDRESS_HOST=localhost
BIND_ADDRESS_PORT=8080
AUTH_DISCOVERY_URI=http://auth.permaplant.net/realms/PermaplanT/.well-known/openid-configuration
AUTH_CLIENT_ID=localhost
```

@absurd-turtle or @kitzbergerg please do the last updates, so that we can finally merge this.